docs/9.x/changelog.md

```diff
@@ -357,6 +357,21 @@ All changes listed are in comparison to 7.0.30 LTS.
 
 ## 7.0 Releases
 
+### 7.0.36 (January 19, 2022)
+
+#### Bugfixes
+
+* Upgrades Go version to 1.17.5 (CVE-2021-44716 / CVE-2021-44717) ([#2698](https://github.com/gravitational/gravity/pull/2698), [planet#873](https://github.com/gravitational/planet/pull/873)).
+
+!!! warning
+    This release fixes a security vulnerability in Go. Please see
+    [Go Announcement for CVE-2021-44716/CVE-2021-44717](https://groups.google.com/g/golang-announce/c/hcmEScgc00k)
+
+#### Internal Changes
+
+* Upgrades kube-rbac-proxy to v0.6.0 ([#2709](https://github.com/gravitational/gravity/pull/2709), [monitoring-app#224](https://github.com/gravitational/monitoring-app/pull/224)).
+* And other minor code cleanup and build improvements.
```

Comment:
I don't think we tend to include statements like this, as it's not really actionable by a customer, and doesn't link to the underlying changes. So I don't think it's needed here or below.

Reply:
@wadells and I started including this entry for non actionable changes, I guess mainly for book keeping. https://github.com/gravitational/gravity/pull/2599#pullrequestreview-724540228

If you feel strongly about not including this entry in the changelogs we can discuss further.